### ISSUE TITLE
Proxmox HASSIO NUT will not install

### ISSUE THREAD

**Issue by xyloidpeople** [hassio-addons/addon-nut#6]
# Problem

NUT component will not install and did not transfer when upgrading from Raspberry Pi HASSIO to Proxmox.  All other components transferred properly using the snapshot.  Adding the repository does not add the component.

> (Why the issue was filed)

I rely on the NUT to monitor power in my house to prevent floods from sump pump failure.

## Expected behavior
It worked great before and now just will not install

> (What you expected to happen)
Someone to help to get it functional again.

## Actual behavior

> (What actually happened)

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
This addon is not production ready and therefore not stable.

Please refrain from using it at this moment, thank you

**Comment by RoosterX** [comment]
I do know this, but thought you wanted to know. 

Anyway I thank you for your answer. :-)

**Comment by xyloidpeople** [comment]
Thanks for the info Franck, I appreciate the reply!  I had NUT working on
my Pi with HASSIO, then when upgraded to a NUC with PROXMOX the entity went
away.   I could not install it from the snap shot or from the github.  I
use this to watch for power outage as I have flooded before and in Canada
right now, we are experiencing lots of snow melt and rain.

I found this one: https://github.com/colindunn/hassio-addons

That seems to be working.  I will upgrade to yours once it is stable.

Maybe some day I will gain enough info to help you guys out.  I have a
software background, but the linux/container thing is pretty new.

Thanks again,

Steven Manzer
